src/js/components/Carousel/__tests__/__snapshots__/Carousel-test.js.snap

```diff
@@ -323,6 +323,7 @@ exports[`Carousel basic 1`] = `
           className="c8"
           disabled={true}
           onBlur={[Function]}
+          onClick={[Function]}
```

Comment:
Usually, when snapshots changed on refactoring it means that the functionality might have changed, will you be willing to take a look at why this snapshot changed? might mean we are doing something different with `onClick()`

Reply:
I explained it in the background context

Reply:
thanks, I'll re-read